peekingduck/pipeline/nodes/model/efficientdet_d04/efficientdet_model.py

```diff
@@ -51,10 +52,7 @@ def __init__(self, config: Dict[str, Any]) -> None:
             self.logger.info(f"---weights downloaded to {weights_dir}.---")
 
         classes_path = model_dir / config["weights"]["classes_file"]
-        self.class_names = {
-            value["id"] - 1: value["name"]
-            for value in json.load(open(classes_path, "r")).values()
-        }
+        self.class_names = json.loads(Path(classes_path).read_text())
```

Comment:
Previously, `self.class_names` gets loaded as a `Dict[int, str]`, where the key indexing is 0-based.

After briefly testing, it looks like the new one-liner loads `self.class_names` as `Dict[str, Dict[str, str]]`, where the key indexing is 1-based for both the main Dict, and the nested value Dict.

I think this change breaks the `detect_ids` to `labels` conversion in the [`postprocess()`](https://github.com/ongtw/PeekingDuck/blob/feat_wildcard_for_obj_det/peekingduck/pipeline/nodes/model/efficientdet_d04/efficientdet_files/detector.py#L97) function?

Reply:
Ugh, apologies, have fixed the bug.